src/php/Command/Test/TestCommand.php

```diff
@@ -73,13 +71,31 @@ public function addRuntimePath(string $namespacePrefix, array $path): self
     public function execute(InputInterface $input, OutputInterface $output): int
     {
         try {
-            /** @var list<string> $paths */
+            /** @var array<string> $paths */
```

Comment:
Actually, I would prefer to keep using `list` for lists of things.
> list: represents continuous, integer-indexed arrays (always start from index zero) like: [“red”, “yellow”, “blue”] 

Check out this page to find more information about the [Modern PHPDoc Annotations](https://suckup.de/2020/02/modern-phpdoc-annotations/).

Reply:
Done